lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -58,12 +59,27 @@ public ProcessJourneyEventHandler(
             IpvSessionService ipvSessionService,
             ConfigService configService,
             ClientOAuthSessionDetailsService clientOAuthSessionService,
-            List<IpvJourneyTypes> journeyTypes)
+            List<IpvJourneyTypes> journeyTypes,
+            StateMachineInitializerMode stateMachineInitializerMode)
             throws IOException {
         this.ipvSessionService = ipvSessionService;
         this.configService = configService;
         this.clientOAuthSessionService = clientOAuthSessionService;
-        this.stateMachines = loadStateMachines(journeyTypes);
+        this.stateMachines = loadStateMachines(journeyTypes, stateMachineInitializerMode);
+    }
+
+    public ProcessJourneyEventHandler(
+            IpvSessionService ipvSessionService,
+            ConfigService configService,
+            ClientOAuthSessionDetailsService clientOAuthSessionService,
+            List<IpvJourneyTypes> journeyTypes)
+            throws IOException {
+        this(
+                ipvSessionService,
+                configService,
+                clientOAuthSessionService,
+                journeyTypes,
+                StateMachineInitializerMode.STANDARD);
```

Comment:
I don't think you need this additional constructor - it's only used in one test?

Reply:
I think there may be a desire to continue to use the test journey map because testing on a fixed journey map can only be so useful.
Is there a way I can mock/ build this without the constructor do you think?

Reply:
As in, you've got two constructors which only differ by the last parameter, and this version is only used in one test. Could you just specify the extra parameter in that test?